### PR TITLE
Homework2: Alexander Alexeyuk

### PR DESCRIPTION
**Pull request by AlexAlexeyuk**
### PR checklist
Please make sure you've passed PR checklist
- [x] my PR passes CI checks (flake8 and pycodestyle)
- [x] my PR named according to guidelines.
- [x] my PR contains complete tasks descriptions for all tasks
- [x] my PR made from the fork feature branch to maxim homework# branch.
- [x] my PR contains only code that solves appropriate homework tasks. Redundant 
files (configs, etc. ) are put to .gitignore

### CODE REVIEW

File: HomeWork_2/task1.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    :param n: копейки
+    :param s: количество товара
+    :return: строка. общая цена в рублях и копейках. формат:
+    'x rubles y kopecks'
+    """
+    m = int(input("Сколько рублей? "))
+    n = int(input("Копеек: "))
+    s = int(input("Введите сколько товара вы хотите: "))
+    price_rub = m*s 
+    price_coins = n*s
```
Comment (belovmd) [inline HomeWork_2/task1.py]: works wrong

**Review by belovmd — COMMENTED** [review]
Thank you for your work. 
Please address next paragraphs from review notes [Review Notes](https://github.com/belovmd/it-academy-summer/blob/master/Homework%20review/Homework2.md):
- 1
- 8
- 6
- 11
- 16
- 18

**Comment by AlexAlexeyuk** [comment]
cannot win insane tests

File: HomeWork_2/task1.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+:param m: рубли
+:param n: копейки
+:param s: количество товара
+:return: строка. общая цена в рублях и копейках.
+формат:'x rubles y kopecks' 
+"""
+
+    m = int(input("Сколько рублей? "))
+    n = int(input("Копеек: "))
+    s = int(input("Введите сколько товара вы хотите: "))
```
Comment (belovmd) [inline HomeWork_2/task1.py:15]: these values are parameters in functions

File: HomeWork_2/task2.py [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+    """Поиск самого длинного слова в предложении.
+
+    :param str_: входная строка
+    :return: строка. Самое длинное слово в предложении (в случае если их
+    несколько, самое левое в строке).
+    в случае если
+    """
+
+    str_ = input("Enter some words: ")
+    return("the longest word is: " + max(str_.strip("!#$%&'()*+, -./:;<=>?@[\]^_`{|}~'").split(), key = len))
```
Comment (belovmd) [inline HomeWork_2/task2.py:12]: doesn't solve task

**Comment by belovmd** [comment]
failed ci checks
few tasks
6

**Pull request closed without merging**